server/storage.lua

```diff
@@ -95,6 +95,14 @@ end
 ---@field position vector4
 ---@field metadata PlayerMetadata
 
+---@class PlayerEntityDatabase : PlayerEntity
+---@field charinfo string
+---@field money string
+---@field job? string
+---@field gang? string
+---@field position string
+---@field metadata string
+
```

Comment:
Why does PlayerEntityDatabase exist? I'm a bit confused.

Reply:
Just so I can type the database call, as the database returns a slightly different result than PlayerEntity is